tests/functional/adapter/dbt_show/test_dbt_show.py

```diff
@@ -1,9 +1,44 @@
+import pytest
 from dbt.tests.adapter.dbt_show.test_dbt_show import BaseShowSqlHeader, BaseShowLimit
 
+from dbt.tests.util import run_dbt
+
+model_with_json_struct = """
+    select *
+    from (
+        select
+  struct<
+    k array<
+        struct<c1 int64, c2 json>
+      >
+  >(
+    [
+      struct(
+        1 as c1,
+        to_json(struct(1 as a)) as c2
+      )
+    ]
+  )
+  as v
+    ) as model_limit_subq
+    limit 5
+    """
+
 
 class TestBigQueryShowLimit(BaseShowLimit):
     pass
 
 
 class TestBigQueryShowSqlHeader(BaseShowSqlHeader):
     pass
+
+
+class TestBigQueryShowSqlWorksWithJSONStruct:
```

Comment:
Our team is trying to get better at tying test cases to features/bugs, though we haven't written that down anywhere. Could you add a comment here that indicates that this test case is addressing this [bug](https://github.com/dbt-labs/dbt-bigquery/issues/972).

Reply:
Good idea 💡

@mikealfare Could you provide a code suggestion of what that comment might look like? If it's a formal GitHub code suggestion, then @matt-winkler (or someone else) could just click a button to add it as a commit.

Reply:
I updated